app/models/User.scala

```diff
@@ -7,24 +7,35 @@ import utils.Hash
 import scala.concurrent.ExecutionContext.Implicits.global
 import java.util.Date
 
-case class User(email: Option[String],
-                passwordHash: Option[String],
+import scala.concurrent.Future
+
+case class User(email: Option[User.Email],
+                passwordHash: Option[User.BcryptPassword],
                 id: String,
-                username: Option[String],
+                username: Option[User.UserName],
                 facebookToken: Option[String],
                 facebookUserId: Option[String],
                 creationDate: Date,
+                // public , private or unlisted. Specifies how the story can be retrieved
```

Comment:
My mistake : "public , private or unlisted" are for privacy in story and it's still in a PR.
This state on User is not used.